code/drasil-example/glassbr/lib/Drasil/GlassBR/Body.hs

```diff
@@ -196,7 +196,7 @@ priorityNFReqs = [correctness, verifiability, understandability,
 
 {--INTRODUCTION--}
 
-startIntro :: NamedChunk -> Sentence -> CI -> Sentence
```

Comment:
If you're having to make changes like this in the examples, you should take the opportunity to generalize it so that it works for any type that satisfies the signature of `asStart` (which ought itself to be class-polymorphic).

Reply:
This part should not have been marked 'resolved' as it is not.